android/uhabits-core/src/main/java/org/isoron/uhabits/core/ui/NotificationTray.java

```diff
@@ -202,23 +202,16 @@ public void onPostExecute()
                 return;
             }
 
-            if (!habit.hasReminder()) {
-                systemTray.log(String.format(
-                        Locale.US,
-                        "Habit %d does not have a reminder. Skipping.",
-                        habit.id));
-                return;
-            }
-
```

Comment:
Could you move this block up (before the `shouldShowReminderToday` block) instead of removing it?